importer/Monument.py

```diff
@@ -148,86 +150,139 @@ def substitute_statement(self, prop_name, value, quals=None, refs=None):
+        """
+        Add an alias in a specific language.
+
+        :param language: code of language, e.g. "fi"
+        :param text: content of the alias
+        """
         base = self.wd_item["aliases"]
         if language not in base:
             base[language] = []
         base[language].append(text)
 
     def add_description(self, language, text):
+        """
+        Add a description in a specific language.
+
+        :param language: code of language, e.g. "fi"
+        :param text: content of the description
+        """
         base = self.wd_item["descriptions"]
         base[language] = text
 
     def set_country(self):
+        """Set the country using the mapping file."""
         code = self.mapping["country_code"].lower()
         country = [item["item"]
                    for item in self.adm0 if item["code"].lower() == code][0]
         self.add_statement("country", country)
 
     def set_is(self):
+        """Set the P31 using the mapping file."""
         default_is = self.mapping["default_is"]
         self.add_statement("is", default_is["item"])
 
     def set_labels(self, language, content):
+        """
+        Add a label in a specific language using content that potentially
+        has markup and has to be cleaned from it.
+
+        :param language: code of language, e.g. "fi"
+        :param text: content of the label
+        """
         self.add_label(language, utils.remove_markup(content))
 
     def set_heritage(self):
+        """Set the heritage status, using mapping file."""
         if "heritage" in self.mapping:
             heritage = self.mapping["heritage"]
             self.add_statement("heritage_status", heritage["item"])
 
     def set_coords(self, coord_keywords_tuple):
+        """
+        Add coordinates.
+
+        :param coord_keywords_tuple: names of the columns with
+            coordinates, like ("lat", "long")
+        """
         lat = coord_keywords_tuple[0]
         lon = coord_keywords_tuple[1]
         if self.has_non_empty_attribute(lat):
             if self.lat == 0 and self.lon == 0:
                 return
             else:
+                latitude = getattr(self, lat)
+                longitude = getattr(self, lon)
                 self.add_statement(
-                    "coordinates", (getattr(self, lat), getattr(self, lon)))
+                    "coordinates", (latitude, longitude))
 
     def set_image(self, image_keyword="image"):
+        """
+        Add image.
+
+        :param image_keyword: the name of the column with image path
+        """
         if self.has_non_empty_attribute(image_keyword):
-            self.add_statement("image", getattr(self, image_keyword),
-                               refs=False)
+            image = getattr(self, image_keyword)
+            self.add_statement("image", image, refs=False)
 
     def set_commonscat(self, keyword="commonscat"):
+        """
+        Add Commons category.
```

Comment:
Do we want to clarify that we are adding the statement (as compared to `set_registrant_url` which sets an internal variable)?